resources/js/features/games/components/GameContentWarnings/GameContentWarnings.tsx

```diff
@@ -0,0 +1,81 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import type { IconType } from 'react-icons/lib';
+import { LuShield, LuView } from 'react-icons/lu';
+import { route } from 'ziggy-js';
+
+import { InertiaLink } from '@/common/components/InertiaLink';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { cn } from '@/common/utils/cn';
+import type { TranslatedString } from '@/types/i18next';
+
+const MATURE_THEME_HUB_ID = 7869;
+const PHOTOSENSITIVE_HUB_ID = 25577;
+
+export const GameContentWarnings: FC = () => {
+  const { hasMatureContent, hubs } = usePageProps<App.Platform.Data.GameShowPageProps>();
+  const { t } = useTranslation();
+
+  const photosensitiveWarningHub = hubs.find(
+    (hub) =>
+      hub.id === PHOTOSENSITIVE_HUB_ID && hub.title?.toLowerCase().includes('photosensitive'),
+  );
+
+  if (!hasMatureContent && !photosensitiveWarningHub) {
+    return null;
+  }
+
+  return (
+    <div
+      data-testid="content-warnings"
+      className="rounded-lg bg-embed light:border light:border-neutral-200 light:bg-white"
+    >
+      <div className="flex flex-col gap-0.5 rounded-lg bg-[rgba(50,50,50,0.3)] light:bg-neutral-50">
+        {hasMatureContent ? (
+          <ContentWarning
+            href={route('hub.show', { gameSet: MATURE_THEME_HUB_ID })}
+            Icon={LuShield}
+            label={t('Mature Content')}
+          />
+        ) : null}
+
+        {photosensitiveWarningHub ? (
+          <ContentWarning
+            href={route('hub.show', { gameSet: PHOTOSENSITIVE_HUB_ID })}
+            Icon={LuView}
+            label={t('Photosensitive Epilepsy Warning')}
+          />
```

Comment:
Since this links to the hub, should it be excluded from the metadata?

<img width="354" height="326" alt="Image" src="https://github.com/user-attachments/assets/f977d45b-4827-4756-9036-36b2af5ae054" />

I guess the mature theme link still appears in the metadata.

<img width="353" height="386" alt="Image" src="https://github.com/user-attachments/assets/1284eb3c-4cb7-46b8-bfa1-e28bf853814e" />